python/setup.py

```diff
@@ -73,6 +74,16 @@ def run(self):
         # TODO(rkn): Fix this.
         subprocess.check_call(["../build.sh", "-p", sys.executable])
 
+        # Copy over the generated ray files from compilation.
+        for filename in ray_files_generated:
+            source = "../build/" + filename
+            destination = "ray/core/" + filename
+            self.move_file(source, os.path.join(self.build_lib, destination))
+            self.move_file(source, destination)
+        # Copy over the existing ray files.
+        for filename in ray_files:
+            self.move_file(filename, os.path.join(self.build_lib, filename))
```

Comment:
Right now, we can do something like

```
cd python/ray/core
make -j8
```

to rebuild the executables, and if we installed Ray from source in development mode, then the new executables will be used by `ray.init()`.

I think this PR changes the workflow so that the new executables will not be used unless the copying above is done, is that right?

Reply:
Yes, that's right. I submit a new commit and solve the problem to make sure `make -j8` can still work as before. Just modify the CMakeLists.txt file.